src/api/firebase.js

```diff
@@ -303,3 +303,42 @@ export async function deleteItem() {
 	 * this function must accept!
 	 */
 }
+
+export function comparePurchaseUrgency(data) {
+	const currentDate = new Date();
+
+	const updatedData = data.map((item) => {
+		// calculate time difference directly without using the absolute value built into getDaysBetweenDates function
+		const daysUntilNextPurchase = Math.floor(
+			(item.dateNextPurchased.toDate() - currentDate) / 86400000,
```

Comment:
Suggestion is to assign day in milliseconds to a const for readability

Reply:
Thanks for the suggestion! We'll update that. :)